### ISSUE TITLE
dtw_Loss

### ISSUE THREAD

**Issue by lsztzp** [DaniMS-ZGZ/ScanGAN360#2]
![image](https://github.com/DaniMS-ZGZ/ScanGAN360/assets/73343106/7d22df31-e59a-42c4-854a-5dc96123b6b8)
Hello author, I'm wondering if the order of the last two dimensions of g_fake_data and scanpath was reversed when training using the SoftDTW loss function, 
I think it should be "g_fake_data = torch.reshape(g_fake_data,(b_size,g_fake_data.shape[-1]//3,3))".
Thank you for your help! sincerely
